packages/@aws-cdk/aws-scheduler-targets-alpha/test/integ.schedule-lambda-target.handler/index.ts

```diff
@@ -6,8 +6,12 @@ exports.handler = async (event: any, _context: any) => {
   // assets index.ts file copied to the cdk - integ.out directory
   const client = new LambdaClient();
 
+  // Skip tagging when passed tag value is missing (3rd case)
+  const tagValue = event.tagValue;
+  if (!tagValue) return;
```

Comment:
Previously, the 3rd case occurs a validation error at `lambda:TagResource` API call. Because, if schedule target input is absent, the `event` argument is an EventBridge event record and the length exceeds the limit of tag value. This change can skip the API call when an assertion is not needed.